---a/net/minecraft/item/crafting/RecipesWeapons.java
+++b/net/minecraft/item/crafting/RecipesWeapons.java
@@ -5,33 +5,29 @@
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
 
-public class RecipesWeapons
-{
-    private String[][] recipePatterns = new String[][] {{"X", "X", "#"}};
+public class RecipesWeapons {
+
+    private String[][] recipePatterns = new String[][]{{"X", "X", "#"}};
     private Object[][] recipeItems;
     private static final String __OBFID = "CL_00000097";
 
-    public RecipesWeapons()
-    {
-        this.recipeItems = new Object[][] {{Blocks.planks, Blocks.cobblestone, Items.iron_ingot, Items.diamond, Items.gold_ingot}, {Items.wooden_sword, Items.stone_sword, Items.iron_sword, Items.diamond_sword, Items.golden_sword}};
+    public RecipesWeapons() {
+        this.recipeItems = new Object[][]{{Blocks.planks, Blocks.cobblestone, Items.iron_ingot, Items.diamond, Items.gold_ingot}, {Items.wooden_sword, Items.stone_sword, Items.iron_sword, Items.diamond_sword, Items.golden_sword}};
     }
 
-    public void addRecipes(CraftingManager p_77583_1_)
-    {
-        for (int i = 0; i < this.recipeItems[0].length; ++i)
-        {
+    public void addRecipes(CraftingManager p_77583_1_) {
+        for (int i = 0; i < this.recipeItems[0].length; ++i) {
             Object object = this.recipeItems[0][i];
 
-            for (int j = 0; j < this.recipeItems.length - 1; ++j)
-            {
-                Item item = (Item)this.recipeItems[j + 1][i];
-                p_77583_1_.addRecipe(new ItemStack(item), new Object[] {this.recipePatterns[j], '#', Items.stick, 'X', object});
+            for (int j = 0; j < this.recipeItems.length - 1; ++j) {
+                Item item = (Item) this.recipeItems[j + 1][i];
+                p_77583_1_.addRecipe(new ItemStack(item), new Object[]{this.recipePatterns[j], '#', Items.stick, 'X', object});
             }
         }
 
-        p_77583_1_.addRecipe(new ItemStack(Items.bow, 1), new Object[] {" #X", "# X", " #X", 'X', Items.string, '#', Items.stick});
-        p_77583_1_.addRecipe(new ItemStack(Items.arrow, 4), new Object[] {"X", "#", "Y", 'Y', Items.feather, 'X', Items.flint, '#', Items.stick});
+        p_77583_1_.addRecipe(new ItemStack(Items.bow, 1), new Object[]{" #X", "# X", " #X", 'X', Items.string, '#', Items.stick});
+        p_77583_1_.addRecipe(new ItemStack(Items.arrow, 4), new Object[]{"X", "#", "Y", 'Y', Items.feather, 'X', Items.flint, '#', Items.stick});
     }
-}
+}
